aiida/orm/implementation/general/code.py

```diff
@@ -454,6 +454,37 @@ def new_calc(self, *args, **kwargs):
         new_calc.use_code(self)
         return new_calc
 
+    def get_new_builder(self):
+        """
+        Create and return a new ProcessBuilder for the default Calculation
+        plugin, as obtained by the self.get_input_plugin_name() method.
+
+        :note: it also sets the ``builder.code`` value.
+
+        :raise MissingPluginError: if the specified plugin does not exist.
+        :raise ValueError: if no default plugin was specified.
+
+        :return:
+        """
+        from aiida.orm.utils import CalculationFactory
+        plugin_name = self.get_input_plugin_name()
+        if plugin_name is None:
+            raise ValueError(
+                "You did not specify a default input plugin for this code")
+        try:
+            C = CalculationFactory(plugin_name)
+        except MissingPluginError:
+            raise MissingPluginError(
+                "The input_plugin name for this code is "
+                "'{}', but it is not an existing plugin"
+                "name".format(plugin_name))
+
+        builder = C.get_builder()
```

Comment:
I am not an expert on this and I don't know if I am the best to review it.
I don't see anywhere in the AiiDA code the .get_builder method. Maybe it is at an external repository.
I suppose that there is somewhere a test that the get_builder returns something meaningful.

Apart from that, the above code is just a wrapper of that procedure and seems OK